packages/react-dev-utils/ModuleScopePlugin.js

```diff
@@ -37,24 +37,28 @@ class ModuleScopePlugin {
       // Maybe an indexOf === 0 would be better?
       const relative = path.relative(appSrc, request.context.issuer);
       // If it's not in src/ or a subdirectory, not our request!
+      if (relative.startsWith('../') || relative.startsWith('..\\')) {
+        return callback();
+      }
+      const requestFullPath = path.resolve(
+        path.dirname(request.context.issuer),
+        request.__innerRequest_request
+      );
+      const requestRelativeToRoot = path.relative(
+        request.descriptionFileRoot,
+        requestFullPath
+      );
```

Comment:
@gaearon you are totally right. My previous code wasn't working . I'm not sure what I did there, but this is what I meant to do - to check this is the package.json that's in the root. I didn't want to assume package.json always lives in `../src` in case the src is more than one level deep like `true/src`